react/features/base/label/components/CircularLabel.native.js

```diff
@@ -155,3 +167,19 @@ export default class CircularLabel extends AbstractCircularLabel<Props, State> {
         }
     }
 }
+
+/**
+ * Maps part of the Redux store to the props of this component.
+ *
+ * @param {Object} state - The Redux store.
+ * @returns {{
+ *     _visible: boolean
+ * }}
+ */
+function _mapStateToProps(state) {
+    return {
+        _visible: !isToolboxVisible(state)
```

Comment:
I don't like this interdependency. The labels aren't visible or not because of the toolbox, it's because of the Navbar. Also, I don't think they should connect to the store to get this information, since these components are relatively generic and could go anywhere. They should just get the prop passed to them.

Reply:
Yeah, a prop would do the trick, instead of this, thanks!

About the dependency, please see my other comment below.